src/cratedb_fivetran_destination/main.py

```diff
@@ -246,9 +240,15 @@ def DescribeTable(self, request, context):
         log_message(LOG_INFO, f"Completed fetching table info: {table}")
         return destination_sdk_pb2.DescribeTableResponse(not_found=False, table=table)
 
+    def WriteHistoryBatch(self, request, context):
+        """
+        TODO: Implement method for real.
+        """
+        return destination_sdk_pb2.WriteBatchResponse(success=True)
+
     def _configure_database(self, url):
         if not self.engine:
-            self.engine = sa.create_engine(url)
+            self.engine = sa.create_engine(url, echo=True)
```

Comment:
`echo=True` should not slip into production.
```suggestion
            self.engine = sa.create_engine(url, echo=False)
```